boards/arm/mimxrt685_evk/Kconfig.defconfig

```diff
@@ -14,6 +14,15 @@ config XTAL_SYS_CLK_HZ
 config SYSOSC_SETTLING_US
 	default 260
 
+config FLASH_MCUX_FLEXSPI_NOR
+	default y if FLASH
+
+config FLASH_MCUX_FLEXSPI_XIP
+	default y
+
+config FLASH_MCUX_FLEXSPI_XIP_MEM
+	default "SRAM"
```

Comment:
Use `config FLASH_MCUX_FLEXSPI_XIP_MEM_TARGET` instead of `config FLASH_MCUX_FLEXSPI_XIP_MEM`. Else `#define CONFIG_FLASH_MCUX_FLEXSPI_XIP_MEM_ITCM 1` is set

Reply:
Done.